internal/bminventory/inventory.go

```diff
@@ -1551,9 +1551,14 @@ func (b *bareMetalInventory) PostStepReply(ctx context.Context, params installer
 	return installer.NewPostStepReplyNoContent()
 }
 
-func handleReplyError(params installer.PostStepReplyParams, b *bareMetalInventory, ctx context.Context, h *models.Host) error {
+func (b *bareMetalInventory) handleReplyError(params installer.PostStepReplyParams, ctx context.Context, h *models.Host) error {
 
 	if params.Reply.StepType == models.StepTypeInstall {
+		// Handle case of installation failure due to an already running assisted-installer.
+		if params.Reply.ExitCode == 137 && strings.Contains(params.Reply.Error, "the container name \"assisted-installer\" is already in use") {
```

Comment:
Interesting if this will work if logs_sender will fail too as we saw?

Reply:
That's why I verify this text in the error log:
"the container name \"assisted-installer\" is already in use"